src/sagemaker/lineage/query.py

```diff
@@ -155,6 +167,19 @@ def __str__(self):
         """
         return str(self.__dict__)
 
+    def __repr__(self):
+        """Define string representation of ``Vertex``.
+
+        Format:
+            {
+                'arn': 'string', 'lineage_entity': 'string',
+                'lineage_source': 'string',
+                '_session': <sagemaker.session.Session object>
+            }
+
+        """
+        return "\n\t" + str(self.__dict__)
```

Comment:
Where does `self.__dict__` come from?

Reply:
It's a built-in class attribute. It'll get the dictionary for the defined attributes

Reply:
__ repr__ of `Vertices` and `Edges` will automatically be called when `LineageQueryResult` __str__ is called (`LineageQueryResult` has lists of `Vertices` and `Edges`), so thats why `\n\t` is added in __repr__ so that when `LineageQueryResult` __str__ is called, it won't be whole chuck of text instead it would contain indentation for `Vertices` and `Edges` and will be easier to read.